L1Trigger/Phase2L1ParticleFlow/src/firmware/pfalgo2hgc.h

```diff
@@ -0,0 +1,46 @@
+#ifndef FIRMWARE_PFALGO2HGC_H
+#define FIRMWARE_PFALGO2HGC_H
+
+#ifndef REG_HGCal
+#ifndef CMSSW_GIT_HASH
+#warning "REG_HGCal is not #defined, but this algorithm has only been tested there"
```

Comment:
no

Reply:
This is for when it's used outside CMSSW when building firmware.
That said, if you feel strongly about it I can remove the warning

Reply:
removed the warning